### PR TITLE
Tim rube/app/cosplay list

### PR DESCRIPTION
**Pull request by violetblokken**

### CODE REVIEW

File: CosNet.AppUI.Android/app/src/main/java/cosnet/android/ui/home/HomeFragment.java [hunk 1]
```diff
@@ -1,35 +1,113 @@
...
-                             ViewGroup container, Bundle savedInstanceState) {
-        homeViewModel =
-                ViewModelProviders.of(this).get(HomeViewModel.class);
-        View root = inflater.inflate(R.layout.fragment_home, container, false);
-        final TextView textView = root.findViewById(R.id.text_home);
+    homeViewModel = ViewModelProviders.of(this).get(HomeViewModel.class);
+    View root = inflater.inflate(R.layout.fragment_home, container, false);
+        /*final TextView textView = root.findViewById(R.id.text_home);
         homeViewModel.getText().observe(getViewLifecycleOwner(), new Observer<String>() {
             @Override
```
Comment (Joren-Thijs) [inline CosNet.AppUI.Android/app/src/main/java/cosnet/android/ui/home/HomeFragment.java]: Why are these comments left?

File: CosNet.AppUI.Android/app/src/main/java/cosnet/android/ui/home/HomeFragment.java [hunk 2]
```diff
@@ -1,35 +1,113 @@
...
+    });
+    addNewCosplayBTN.setOnClickListener(v -> {
+      Intent intent = new Intent(getActivity(), AddCosplay.class);
+      startActivity(intent);
+    });
+
+    return root;
+  }
+
+  class CosplayListAdapter extends ArrayAdapter<Cosplay> {
```
Comment (Joren-Thijs) [inline CosNet.AppUI.Android/app/src/main/java/cosnet/android/ui/home/HomeFragment.java]: Why are there 2 classes in the same file?

File: CosNet.AppUI.Android/app/src/main/java/cosnet/android/ui/home/HomeViewModel.java [hunk 3]
```diff
@@ -6,14 +6,14 @@
 
 public class HomeViewModel extends ViewModel {
 
-    private MutableLiveData<String> mText;
+    //private MutableLiveData<String> mText;
 
     public HomeViewModel() {
-        mText = new MutableLiveData<>();
-        mText.setValue("This is home fragment");
+        //mText = new MutableLiveData<>();
```
Comment (Joren-Thijs) [inline CosNet.AppUI.Android/app/src/main/java/cosnet/android/ui/home/HomeViewModel.java]: Why is all this commented out?

**Review by Joren-Thijs — CHANGES_REQUESTED** [review]
Looks good except some commented out code.
Commented out code should not remain for later. Thats why we have git.

File: CosNet.AppUI.Android/app/src/main/java/cosnet/android/ui/home/HomeFragment.java [hunk 4]
```diff
@@ -1,35 +1,113 @@
...
+    private Context context;
+    int resource;
+
+
```
Comment (kaiye13) [inline CosNet.AppUI.Android/app/src/main/java/cosnet/android/ui/home/HomeFragment.java]: wit regel te veel

File: CosNet.AppUI.Android/app/src/main/res/layout/fragment_home.xml [hunk 5]
```diff
@@ -4,19 +4,57 @@
...
+    app:layout_constraintBottom_toBottomOf="parent"
+    app:layout_constraintEnd_toEndOf="parent"
+    app:layout_constraintStart_toStartOf="parent"
+    app:layout_constraintTop_toBottomOf="@+id/addNewCosplayBTN">
+
+    <ListView
+      android:id="@+id/CosplayList"
+      android:layout_width="match_parent"
+      android:layout_height="match_parent">
+
```
Comment (kaiye13) [inline CosNet.AppUI.Android/app/src/main/res/layout/fragment_home.xml]: witregeltje

**Review by kaiye13 — CHANGES_REQUESTED** [review]

**Review by Joren-Thijs — APPROVED** [review]

**Review by kaiye13 — APPROVED** [review]

**Pull request merged**